CONTRIBUTING.md

```diff
@@ -1,9 +1,10 @@
 # Contributing to Node.js
 
-* [Code of Conduct](#code-of-conduct)
-* [Issues](#issues)
-* [Pull Requests](#pull-requests)
-* [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin)
+- [Contributing to Node.js](#contributing-to-nodejs)
+  - [Code of Conduct](#code-of-conduct)
+  - [Issues](#issues)
+  - [Pull Requests](#pull-requests)
+  - [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
```

Comment:
As with the change in the other file, I'm pretty sure this will fail linting and should use asterisks instead for consistency with our other files.